### ISSUE TITLE
NavigationView - Android: screen will zoom in on the first leg of the multi stop route on Android when navigationViewController.showOverview is called.

### ISSUE THREAD

**Issue by rhinkle** [googlemaps/react-native-navigation-sdk#337]
## Description
When route is initialized the screen will zoom in on the first leg of the multi stop route on Android. iOS zooms to the level of the full route is visible on the screen.

### Environment details

**Library version and other environment information:**

- react-native-navigation-sdk version: 0.6.1

**OS type and version**

- iOS: 18.1, 18.0
- Android: 35

## Steps to reproduce
1. Pull down this branch. 
2. Install the dependencies. 
3. Run yarn start 
4. Run yarn ios or yarn android
5. Click on navigate button
6. Once the screen is done loading you should see the map zoomed in on the first leg of the route for android.

### Code example
[Link to code example](https://github.com/rhinkle/react-native-navigation-sdk/tree/ISSUE-AndroidZoomOnAfterRouteSet)
[Link to onRouteOk](https://github.com/rhinkle/react-native-navigation-sdk/blob/ISSUE-AndroidZoomOnAfterRouteSet/example/src/NavigationScreen.tsx#L204-L207)

![Screenshot 2024-11-13 at 2 52 11 PM](https://github.com/user-attachments/assets/7023ee36-777e-43ae-bc54-0506fb39cbc5)

**Comment by jokerttu** [comment]
Hi @rhinkle 
Behaviour differs in underlaying Navigation SDK's for Android and iOS.
As a workaround support to animate camera to bounds should be implemented.
This difference need to be documented to the `showOverview` method.

**Comment by jokerttu** [comment]
This is actually duplicate of the #171

**Comment by illuminati1911** [comment]
Closing the issue as it is a duplicate.
